src/Turing.jl

```diff
@@ -68,6 +56,15 @@ using .ModeEstimation
     export optimize
 end
 
+if !isdefined(Base, :get_extension)
+    using Requires
+end
+function __init__()
+    @static if !isdefined(Base, :get_extension)
```

Comment:
This should be moved outside of the `__init__` function to avoid defining it when the default is sufficient.

Reply:
Like this?(See last commit)